nibabel/orientations.py

```diff
@@ -322,7 +322,7 @@ def axcodes2ornt(axcodes, labels=None):
            [ 2.,  1.]])
     """
     labels = list(zip('LPI', 'RAS')) if labels is None else labels
-    allowed_labels = sum([list(L) for L in labels], []) + [None]
+    allowed_labels = sum((list(L) for L in labels), []) + [None]
```

Comment:
It took me some thinking to understand what this line does. Isn't this code abusing `sum()`? It should at least be moved to a function named `flatten()` or something similar.

Alternatively:
```
    allowed_labels = list(itertools.chain.from_iterable((list(L) for L in labels)))
````
```
    allowed_labels = list(functools.reduce(operator.add, (list(L) for L in labels), []))
````

Reply:
`sum(list_of_lists, [])` is pretty standard, but this definitely isn't a pretty way of going about it.

If we want to normalize sequences to lists and add (with the marginal benefit of using only builtins):

```suggestion
    allowed_labels = sum(map(list, labels), [None])
```

If we want to chain values and put them in a list at the end:

```suggestion
    allowed_labels = list(chain.from_iterable(labels)) + [None]
```

Definitely don't need to do both.